### PR TITLE
Utilization: Add Disk Usage chart

### PR DESCRIPTION
**Pull request by mareklibra**
and refactor prometheus queries for disk capacity so they
can be reused for utilization.

Depends on: https://github.com/kubevirt/web-ui-components/pull/322

### CODE REVIEW

**Comment by mareklibra** [comment]
Ceph metrics aligned with https://github.com/kubevirt/web-ui/pull/273

File: frontend/public/kubevirt/components/cluster/cluster-overview.jsx [hunk 1]
```diff
@@ -30,17 +31,18 @@ const OPENSHIFT_VERSION_QUERY = 'openshift_build_info{job="apiserver"}';
 
 const CAPACITY_MEMORY_TOTAL_QUERY = 'sum(kube_node_status_capacity_memory_bytes)';
 
-const CAPACITY_STORAGE_TOTAL_QUERY = 'sum(kubelet_volume_stats_capacity_bytes)'; // available with Ceph
-const CAPACITY_STORAGE_USED_QUERY = 'sum(kubelet_volume_stats_used_bytes)';
-const CAPACITY_STORAGE_TOTAL_DEFAULT_QUERY = 'sum(node_filesystem_avail_bytes)';
-const CAPACITY_STORAGE_USED_DEFAULT_QUERY = 'sum(node_filesystem_avail_bytes) - sum(node_filesystem_free_bytes)';
+const CAPACITY_STORAGE_TOTAL_QUERY = 'max_over_time(ceph_cluster_total_bytes[60m:])'; // available with Ceph
```
Comment (rawagner) [inline frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: I dont think this is correct - we want latest value of storage capacity, not max in last 60minutes. You can get wrong result if capacity goes down in the given timeframe.

Comment (rawagner) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: it should be just `ceph_cluster_total_bytes` and `sum(node_filesystem_avail_bytes)` IMO

Comment (mareklibra) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: Actually this is different for Capacity and Utilization.
For Capacity, you are right - it shows the "actual". 
For utilization, we need to use the maximum of the time period as the last value can be lower than a former one.

I will fix that for the Capacity flow.

Comment (mareklibra) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: But we are lucky, we do not need to deal with that explicitly as there is the search for maximal past value multiplied by 2. So we should be fine even with the chart.

Comment (mareklibra) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: done

File: frontend/public/kubevirt/components/cluster/cluster-overview.jsx [hunk 2]
```diff
@@ -163,14 +166,42 @@ export class ClusterOverview extends React.Component {
...
   }
 
-  getPrometheusMetrics() {
+  async _getStorageMetrics() {
+    try {
+      let queryTotal = CAPACITY_STORAGE_TOTAL_DEFAULT_QUERY;
+      let queryUsed = UTILIZATION_STORAGE_USED_DEFAULT_QUERY;
+
+      const metrics = await this.getPrometheusMetrics();
+      if (_.get(metrics, 'data', []).find(metric => metric === 'kubelet_volume_stats_capacity_bytes')) {
```
Comment (rawagner) [inline frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: is this check correct ? Its a metric that was used before. We should rather check for presence of one of new ceph metrics right ?

Comment (mareklibra) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: oh, good catch. I'll parametrize that to avoid similar situation in the future

**Pull request merged**